includes/Experiments.php

```diff
@@ -372,6 +372,18 @@ public function get_experiments(): array {
 				'description' => __( 'Enable improved autosaves support', 'web-stories' ),
 				'group'       => 'editor',
 			],
+			/**
+			 * Author: @timarney
+			 * Issue: #12093
+			 * Creation date: 2022-08-18
+			 */
+			[
+				'name'        => 'cropHidden',
+				'label'       => __( 'Crop Hidden', 'web-stories' ),
```

Comment:
Nit pick, but can we rename this here and everywhere else? "Crop hidden" does not really convey what this does.

Maybe "offScreenVideoCropping" / "Crop off-screen video parts" or something.